Sealnote/src/main/java/com/twistedplane/sealnote/NoteActivity.java

```diff
@@ -532,8 +581,9 @@ public void doRestore() {
      * Called when save action is invoked by click
      */
     public void doSave() {
-        if (mSaveButtonClicked) return; else mSaveButtonClicked = true; //FIXME: Hack. Avoids double saving
-        final boolean noteSaved = saveNote();
+        if (mSaveButtonClicked) return;
+        else mSaveButtonClicked = true; //FIXME: Hack. Avoids double saving
+        final boolean noteSaved = saveNote(false);
```

Comment:
Since `noteSaved` is used just once you can inline it. I wrote the original code, but since you are changing these lines, lets clean this little part as well.